### ISSUE TITLE
Upgrade to react-navigation 5.X

### ISSUE THREAD

**Issue by tyrauber** [lezzmo/razzle-treact-natigation#7]
@lezzmo, Thanks for this.

What do you think about upgrading to [react-navigation 5.X](https://reactnavigation.org/docs/getting-started)?  This example is currently using the 3.X syntax, specifically `createAppContainer` which is no longer used in 5.x, replaced with the `NavigatorContainer` Syntax.

```
import { NavigationContainer } from '@react-navigation/native';
import { createStackNavigator } from '@react-navigation/stack';

const Stack = createStackNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Stack.Navigator>{/* ... */}</Stack.Navigator>
    </NavigationContainer>
  );
}
```

[Upgrading from 4.x](https://reactnavigation.org/docs/upgrading-from-4.x/#navigation-container)

Any reason not to keep this current?

**Comment by lesmo** [comment]
Yeah I totally agree.. I believe at the time I built this the `react-navigation` lib was in beta. I'll try to update the lib.
